### PR TITLE
CompiledModuleLoader changes

### PR DESCRIPTION
**Pull request by SreeniIO**
see #56

### CODE REVIEW

**Comment by andrieshiemstra** [comment]
could you alter cargo.toml to point at github/hirofa?

**Comment by SreeniIO** [comment]
> could you alter cargo.toml to point at github/hirofa?

Sure, will do. I guess the utils have to be merged before that, otherwise, I'm getting compilation errors while checking in.

File: src/facades.rs [hunk 1]
```diff
@@ -1015,10 +1015,10 @@ pub mod tests {
     #[test]
     pub fn test_stack_size() {
         let rt = init_test_rt();
-        // 150 is ok, 200 fails
+        // 124 is ok, 200 fails
         let res = rt.eval_sync(Script::new(
             "stack_test.js",
-            "let f = function(a){let f2 = arguments.callee; if (a < 150) {f2(a + 1);}}; f(1);",
+            "let f = function(a){let f2 = arguments.callee; if (a < 124) {f2(a + 1);}}; f(1);",
```
Comment (SreeniIO) [inline src/facades.rs:1021]: I had to change to 124 as I was getting stack overflow for any value > 124.

Comment (andrieshiemstra) [reply src/facades.rs:1021]: Yeah that one wasn't an exact science 🙂

**Pull request merged**